pkg/kubelet/status/manager.go

```diff
@@ -434,13 +435,19 @@ func (m *manager) needsReconcile(uid types.UID, status api.PodStatus) bool {
 		pod = mirrorPod
 	}
 
-	if isStatusEqual(&pod.Status, &status) {
+	podStatus, err := copyStatus(&pod.Status)
+	if err != nil {
+		return false
+	}
+	normalizeStatus(&podStatus)
```

Comment:
The statuses from the apiserver don't need to be normalized because they have limited time precision. It's the internal status kubelet generates that need to be normalized. It doesn't hurt to do it again, of course.